dev.exs

```diff
@@ -1246,7 +1265,8 @@ Task.start(fn ->
     Demo.Repo,
     {Phoenix.PubSub, [name: Demo.PubSub]},
     {Beacon, sites: [dev_site, dy_site]},
```

Comment:
should these sites get their own endpoint now instead of using `DemoWeb.Endpoint`?

Reply:
Ideally yes to simulate the same config used in a real app. I created the Proxy endpoint but forgot to create the site endpoints.